clock_exercise/clockmanager.h

```diff
@@ -7,40 +7,72 @@ class ClockManager : public QObject
 {
     Q_OBJECT
 
+    // ↓↓↓ properties
     Q_PROPERTY(QString dateText MEMBER m_dateText NOTIFY dateTextChanged)
     Q_PROPERTY(int minutes MEMBER m_minutes NOTIFY minutesChanged)
     Q_PROPERTY(int hours MEMBER m_hours NOTIFY hoursChanged)
-    Q_PROPERTY(int timeOfTheDissolveAnimation MEMBER TIME_OF_THE_DISSOLVE_ANIMATION CONSTANT)
     Q_PROPERTY(bool timerRunning MEMBER m_timerRunning NOTIFY timerRunningChanged)
     Q_PROPERTY(float timerCurrentSeconds MEMBER m_timerCurrentSeconds NOTIFY timerCurrentSecondsChanged WRITE setTimerCurrentSeconds)
+    Q_PROPERTY(int dayOfDate MEMBER m_dayOfDate NOTIFY dayOfDateChanged WRITE setDayOfDate)
+    Q_PROPERTY(int monthOfDate MEMBER m_monthOfDate NOTIFY monthOfDateChanged WRITE setMonthOfDate)
+    Q_PROPERTY(int yearOfDate MEMBER m_yearOfDate NOTIFY yearOfDateChanged WRITE setYearOfDate)
+    Q_PROPERTY(QString dayOfDateName MEMBER m_dayOfDateName NOTIFY dayOfDateNameChanged WRITE setDayOfDateName)
+    Q_PROPERTY(QString monthOfDateName MEMBER m_monthOfDateName NOTIFY monthOfDateNameChanged WRITE setMonthOfDateName)
```

Comment:
Tutte queste property possono essere ridotte ad una sola:
```
Q_PROPERTY(QDate alarmDate ...)
```
La classe `QDate` ti incapsula di già tutte le informazioni su giorno, mese, anno; ha i metodi `addDays()` etc per modificarla; si trasforma automaticamente in una classe `Date` del Javascript, che puoi quindi utilizzare per estrarre nome del giorno e del mese.

Ti eviti così di avere mille mila oggetti da tenere sincronizzati a mano.

NOTA: Per quello che dicevo prima, il nome del mese non ti serve.